canvas_manage_course/requirements/base.txt

```diff
@@ -1,17 +1,22 @@
 boto3==1.4.4
-Django==1.9.12
-cx-Oracle==5.2.1
+Django==1.11.9
+cx-Oracle==6.0.3
 django-cached-authentication-middleware==0.2.1
 django-redis-cache==1.7.1
 hiredis==0.2.0
 kitchen==1.2.4
 ndg-httpsclient==0.4.2
```

Comment:
Would you mind removing the `ndg-httpsclient` library?    We don't need it on Xenial.